pkgs/by-name/bl/bleep/package.nix

```
@@ -0,0 +1,79 @@
+{
+  stdenvNoCC,
+  fetchzip,
+  autoPatchelfHook,
+  installShellFiles,
+  makeWrapper,
+  lib,
+  zlib,
+}:
+let
+  platform =
+    {
+      x86_64-linux = "x86_64-pc-linux";
+      x86_64-darwin = "x86_64-apple-darwin";
+      aarch64-darwin = "arm64-apple-darwin";
+    }
+    ."${stdenvNoCC.system}" or (throw "unsupported system ${stdenvNoCC.hostPlatform.system}");
+
+  hash =
+    {
+      x86_64-linux = "sha256-dB8reN5rTlY5czFH7BaRya7qBa6czAIH2NkFWZh81ek=";
+      x86_64-darwin = "sha256-tpUcduCPCbVVaYZZOhWdPlN6SW3LGZPWSO9bDStVDms=";
+      aarch64-darwin = "sha256-V8QGF3Dpuy9I6CqKsJRHBHRdaLhc4XKZkv/rI7zs+qQ=";
+    }
+    ."${stdenvNoCC.system}" or (throw "unsupported system ${stdenvNoCC.hostPlatform.system}");
+  version = "0.0.7";
+in
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "bleep";
+  version = version;
+
+  src = fetchzip {
+    url = "https://github.com/oyvindberg/bleep/releases/download/v${version}/bleep-${platform}.tar.gz";
+    sha256 = hash;
+  };
+
+  nativeBuildInputs = [
+    installShellFiles
+    makeWrapper
+  ] ++ lib.optional stdenvNoCC.isLinux autoPatchelfHook;
+
+  buildInputs = [ zlib ];
+
+  installPhase = ''
+    runHook preInstall
+    install -Dm755 bleep -t $out/bin/
+    runHook postInstall
+  '';
+
+  dontAutoPatchelf = true;
+
+  postFixup =
+    lib.optionalString stdenvNoCC.isLinux ''
+      autoPatchelf $out
+    ''
+    + ''
+      mkdir temp
+      cp $out/bin/bleep temp/bleep
+      PATH="./temp:$PATH"
+
+      installShellCompletion --cmd bleep \
+        --bash <(bleep install-tab-completions-bash --stdout) \
+        --zsh <(bleep install-tab-completions-zsh --stdout) \
```

```suggestion
      installShellCompletion --cmd bleep \
        --bash <($out/bin/bleep install-tab-completions-bash --stdout) \
        --zsh <($out/bin/bleep install-tab-completions-zsh --stdout) \
```

---

Also needs `bleep` in `$PATH`

```suggestion
      export PATH=$PATH:$out/bin
      installShellCompletion --cmd bleep \
        --bash <(bleep install-tab-completions-bash --stdout) \
        --zsh <(bleep install-tab-completions-zsh --stdout) \
```

---

Also needs `bleep` in `$PATH`

```suggestion
      export PATH=$PATH:$out/bin
      installShellCompletion --cmd bleep \
        --bash <(bleep install-tab-completions-bash --stdout) \
        --zsh <(bleep install-tab-completions-zsh --stdout) \
```